dvc/command/version.py

```diff
@@ -129,7 +128,12 @@ def get_supported_remotes():
             if not tree_cls.get_missing_deps():
                 supported_remotes.append(tree_cls.scheme)
 
-        return ", ".join(supported_remotes)
+        if len(supported_remotes) == len(TREES):
+            return "All remotes"
+        elif len(supported_remotes) == 1:
+            return supported_remotes
+        else:
+            return ", ".join(supported_remotes)
```

Comment:
nitpicky, but you don't need `elif`/`else` here
```suggestion
        if len(supported_remotes) == 1:
            return supported_remotes
        return ", ".join(supported_remotes)
```